scripts/actions/validate-chainspecs.sh

```diff
@@ -1,27 +1,37 @@
 #!/bin/bash
 set -e # fail on any error
 set -u # treat unset variables as error
-echo "________Running validate_chainspecs.sh________"
-
+echo -e "\n*** Running \`validate_chainspecs.sh\`"
```

Comment:
empty line here is not needed because it is the first printout but fine either way.

```suggestion
echo -e "*** Running \`validate_chainspecs.sh\`"
```